overrides/partials/nav-item.html

```diff
@@ -0,0 +1,66 @@
+{#-
```

Comment:
Note to reviewers: see the .patch files later in the review to see what parts of this file I modified vs. left the same as the upstream version.